scripts/mock_data/supplementary_dataset_metadata_response.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "survey_id": "123",
+    "period_id": "202301",
+    "title": "Supplementary data without repeating items",
+    "sds_schema_version": 1,
+    "sds_published_at": "2022-03-01T12:30:02Z",
+    "total_reporting_units": 2,
+    "schema_version": "v1.0.0",
+    "sds_dataset_version": 1,
+    "filename": "",
+    "dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178"
+  },
+  {
+    "survey_id": "123",
+    "period_id": "202302",
+    "title": "Supplementary data with repeating items",
+    "sds_schema_version": 1,
+    "sds_published_at": "2022-03-01T12:30:02Z",
+    "total_reporting_units": 2,
+    "schema_version": "v1.0.0",
+    "sds_dataset_version": 2,
```

Comment:
The sds_dataset_version would only be 2 if it was the same period. So either set all to same period, or set all version to 1 given period is all diff.